blocks/math.js

```diff
@@ -82,12 +82,19 @@ Blockly.Blocks['math_positive_number'] = {
    * @this Blockly.Block
    */
   init: function() {
-    this.setHelpUrl(Blockly.Msg.MATH_NUMBER_HELPURL);
-    this.setColour(Blockly.Blocks.math.HUE);
-    this.appendDummyInput()
-        .appendField(new Blockly.FieldNumber('0', 0, Infinity), 'NUM');
-    this.setOutput(true, 'Number');
-    this.setOutputShape(Blockly.OUTPUT_SHAPE_ROUND);
+    this.jsonInit({
+      "message0": "%1",
+      "args0": [
+        {
+          "type": "field_number",
+          "name": "NUM",
+          "min": 0
```

Comment:
I made a couple of test projects.. but I'm a bit color blind, so it's hard for me to be much help here..

Can you see a transition from one color to another in the projects? It (shade or color) only goes up by 1 each transition so it's probably hard to see anyways.

https://scratch.mit.edu/projects/115972279/
https://scratch.mit.edu/projects/115972387/